src/bin/cli.js

```diff
@@ -79,18 +78,17 @@ program
 program
   .command('sync')
   .description('Synchronize the local filesystem and the remote cozy')
-  .option('-k, --insecure', 'Turn off HTTPS certificate verification.')
```

Comment:
Probably, but it is a priority for the short term, and the code we had for that will no longer work